platform-validation/scripts/analyzeSystemStats.sh

```diff
@@ -190,8 +190,12 @@ while [[ $SECONDS -lt $end_time ]]; do
                     echo "$binary-size=$file_size" >>"$FILE"
                fi
 
-               memory=$(top -b -d 1 -n 1 | awk '{print $6, $9, $NF}' | grep "$binary$" | awk '{print $1}')
-               cpu=$(top -b -d 1 -n 1 | awk '{print $6, $9, $NF}' | grep "$binary$" | awk '{print $2}')
+               #Top doesn't seem to get Memory CPU in 1ES Agents, Use PS command to get RSS Memory and CPU%
```

Comment:
what was the issue with 1ES agent? was it permission type

Reply:
I have not been able to figure that out yet, top simply returned an empty string for aziot-edge binaries

See example : https://dev.azure.com/msazure/One/_build/results?buildId=53280437&view=logs&j=f1061aa8-bee6-5103-5b5c-2b2c91b98c58&t=de96706b-76c3-5e1c-0eaf-173223e4dad3&l=601

Reply:
Thats weird. Anyways everything looks good. we can test them